docs/src/global/componentApi.jsx

```diff
@@ -0,0 +1,96 @@
+import {
+    camelCase,
+    isEmpty,
+    isNil,
+    map,
+    upperFirst,
+} from 'lodash';
+// eslint-disable-next-line import/extensions
+import makeStyles from 'react-cm-ui/styles/makeStyles';
+import PropTypes from 'prop-types';
+import React from 'react';
+import Heading from './heading';
+import MarkdownContainer from './markdownContainer';
+import ComponentApiTable from './componentApiTable';
+import { DOCS_PROPS_PROP_TYPE } from './componentApiConstants';
+
+const propTypes = {
+    docs: PropTypes.arrayOf(
+        PropTypes.shape({
+            displayName: PropTypes.string,
+            props: DOCS_PROPS_PROP_TYPE,
+        }),
+    ),
+};
+
+const defaultProps = {
+    docs: [],
+};
+
+const useStyles = makeStyles({
+    root: {},
+    tableContainer: {
+        margin: '33px -22px 0',
+    },
+});
+
+function ComponentApi(props) {
+    const {
+        docs,
+    } = props;
+    const classes = useStyles();
+    const isDocsNotDefined = isNil(docs) || isEmpty(docs);
```

Comment:
I think `isEmpty()` covers `isNil()` (i.e. it will return `true` for `undefined` or `null` as well as empty strings and arrays and objects) ... so we can probably just use `isEmpty()`.

Reply:
👍

Reply:
Confirmed this is fixed in 1c3b162.  Resolving.